File: tools-manage/invocationlab-admin/src/main/java/org/javamaster/invocationlab/admin/model/redis/ConnectionVo.java
package org.javamaster.invocationlab.admin.model.redis;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

/**
 * @author yudong
 */
@Builder
@Data
@NoArgsConstructor
@AllArgsConstructor
public class ConnectionVo {
    private String connectId;
    private String nodes;
    private String name;
    private String host;
    private Integer port;
    private String user;
    private String password;
    private Long createTime;
}

File: tools-manage/invocationlab-admin-21/src/main/kotlin/org/javamaster/invocationlab/admin/model/redis/ConnectionVo.kt
package org.javamaster.invocationlab.admin.model.redis

import org.javamaster.invocationlab.admin.annos.AllOpen

/**
 * @author yudong
 */


@AllOpen
class ConnectionVo {
    var connectId: String? = null
    var nodes: String? = null
    var name: String? = null
    var host: String? = null
    var port: Int? = null
    var user: String? = null
    var password: String? = null
    var createTime: Long? = null
}
